std_msgs/Header header
geometry_msgs/Pose pose
int32[] indices
float32[] w_p
float32[] s_p

float32[] si_q_xy
float32[] si_q_theta_Rho
float32[] si_q_xy_img_frame

float32[] si_q_xy_noise
float32[] si_q_theta_Rho_noise
float32[] si_q_xy_img_frame_noise
int32 timestep
